extensions/nbnhhsh/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "nbnhhsh",
+  "title": "nbnhhsh",
+  "description": "nbnhhsh raycast script",
+  "icon": "command-icon.png",
+  "author": "Nolan",
+  "categories": [
+    "Documentation",
+    "Fun"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "nbnhhsh",
+      "subtitle": "便于「土狗」与世界接轨，理解世界",
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.